pandas/_libs/hashtable.pyx

```diff
@@ -59,16 +59,12 @@ include "hashtable_func_helper.pxi"
 # map derived hash-map types onto basic hash-map types:
 if np.dtype(np.intp) == np.dtype(np.int64):
     IntpHashTable = Int64HashTable
-    value_count_intp = value_count_int64
-    duplicated_intp = duplicated_int64
-    ismember_intp = ismember_int64
-    mode_intp = mode_int64
+    unique_label_indices = _unique_label_indices_int64
+    unique_label_indices_intp = _unique_label_indices_int64
 elif np.dtype(np.intp) == np.dtype(np.int32):
     IntpHashTable = Int32HashTable
-    value_count_intp = value_count_int32
-    duplicated_intp = duplicated_int32
-    ismember_intp = ismember_int32
-    mode_intp = mode_int32
+    unique_label_indices = _unique_label_indices_int64
+    unique_label_indices_intp = _unique_label_indices_int32
```

Comment:
value_count_intp = value_count_int32 & Co no longer needed, now there is just value_count which works as supposed to work.